src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('Roboto-Medium.ttf');
+}
+
+html {
+  --brand-color: #00acdc;
```

Comment:
Create a variable with root selector

Reply:
Added new :root selector to store variables there.